src/AlphaLinks.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default function AlphaLinks({ letter }) {
+    return(
+      <li><Link to={`/characters/${letter}`}>{letter}</Link></li>
```

Comment:
yeah, check it out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals